perl/bin/bystro-server.pl

```diff
@@ -53,11 +53,11 @@
 
 # The properties that we accept from the worker caller
 my %requiredForAll = (
-  output_file_base => 'output_base_path',
```

Comment:
this is necessary because currently we don't use un-camelization in perl; I will introduce it later as we begin to convert perl to snake_case